### PR TITLE
fix: chatgpt web

### PR DESCRIPTION
**Pull request by BlackStar1453**
I have fixed chatgpr-web and tested it for local usage. Please verify the usability. 

(Just disregard the previous commits as those were from my forked repository. You only need to review the code changes in the file.)

### CODE REVIEW

File: src/common/services/stream-async-iterable.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+export async function* streamAsyncIterable<T = unknown>(stream: ReadableStream<T>) {
```
Comment (yetone) [inline src/common/services/stream-async-iterable.ts:1]: ‌‌‌It seems like this function isn't used anywhere?Can this file be deleted now?

**Comment by yetone** [comment]
```bash
pnpm lint --fix
```

may help yiou

File: src/common/services/stream-async-iterable.ts (hunk 1 above)
Comment (BlackStar1453) [reply src/common/services/stream-async-iterable.ts:1]: Yes, you can delete it.

Comment (BlackStar1453) [reply src/common/services/stream-async-iterable.ts:1]: You can also delete reference of api.js  in Manifest, this script is no longer needed to generate the arkoseToken.

**Pull request merged**